inference-engine/tests/functional/plugin/cpu/single_layer_tests/space_to_batch.cpp

```diff
@@ -2,77 +2,98 @@
+            for (const auto& item : shape.second) {
+                result << CommonTestUtils::vec2str(item) << "_";
+            }
+        }
+        result << "blockShape=" << CommonTestUtils::vec2str(blockShape) << "_";
+        result << "padsBegin=" << CommonTestUtils::vec2str(padsBegin) << "_";
+        result << "padsEnd=" << CommonTestUtils::vec2str(padsEnd) << "_";
+        result << "netPRC=" << netPrecision.name() << "_";
         result << CPUTestsBase::getTestCaseName(cpuParams);
-
         return result.str();
     }
 
 protected:
     void SetUp() override {
-        LayerTestsDefinitions::spaceToBatchParamsTuple basicParamsSet;
-        CPUSpecificParams cpuParams;
-        std::tie(basicParamsSet, cpuParams) = this->GetParam();
+        targetDevice = CommonTestUtils::DEVICE_CPU;
 
+        std::vector<InputShape>  inputShapes;
+        std::vector<int64_t> blockShape, padsBegin, padsEnd;
+        Precision netPrecision;
+        CPUSpecificParams cpuParams;
+        std::tie(inputShapes, blockShape, padsBegin, padsEnd, netPrecision, cpuParams) = this->GetParam();
         std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
 
-        std::vector<size_t> inputShape;
-        std::vector<int64_t> blockShape, padsBegin, padsEnd;
-        InferenceEngine::Precision netPrecision;
-        std::tie(blockShape, padsBegin, padsEnd, inputShape, netPrecision, inPrc, outPrc, inLayout, outLayout, targetDevice) = basicParamsSet;
-        inPrc = outPrc = netPrecision;
+        auto ngPrec = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+        inType = outType = ngPrec;
+
+        init_input_shapes(inputShapes);
 
         if (strcmp(netPrecision.name(), "U8") == 0)
             selectedType = std::string("ref_any_") + "I8";
         else
             selectedType = std::string("ref_any_") + netPrecision.name();
 
-        auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
-        auto params = ngraph::builder::makeParams(ngPrc, {inputShape});
-        auto paramOuts = ngraph::helpers::convert2OutputVector(
-                ngraph::helpers::castOps2Nodes<ngraph::op::Parameter>(params));
-        auto s2b = ngraph::builder::makeSpaceToBatch(paramOuts[0], ngPrc, blockShape, padsBegin, padsEnd);
-        s2b->get_rt_info() = getCPUInfo();
-        ngraph::ResultVector results{std::make_shared<ngraph::opset1::Result>(s2b)};
+        auto params = ngraph::builder::makeDynamicParams(ngPrec, {inputDynamicShapes.front()});
+        auto paramOuts = ngraph::helpers::convert2OutputVector(ngraph::helpers::castOps2Nodes<ngraph::op::Parameter>(params));
+        auto b2s = ngraph::builder::makeSpaceToBatch(paramOuts[0], ngPrec, blockShape, padsBegin, padsEnd);
+        b2s->get_rt_info() = getCPUInfo();
+        ngraph::ResultVector results{std::make_shared<ngraph::opset1::Result>(b2s)};
         function = std::make_shared<ngraph::Function>(results, params, "SpaceToBatch");
```

Comment:
the same comments: https://github.com/openvinotoolkit/openvino/pull/8639#discussion_r753651630